server/matchmaker/search.py

```diff
@@ -40,8 +42,30 @@ def __init__(self, player, start_time=None, rating_prop='ladder_rating'):
             0: 0.8
         }
 
+    @property
+    def adjusted_rating(self):
+        """
+        Returns an adjusted mean with a simple linear interpolation between current mean and a specified base mean
+        """
+        if self.rating_prop == 'ladder_rating':
+            mean, dev = self.player.ladder_rating
+            adjusted_mean = ((config.NEWBIE_MIN_GAMES - self.player.numGames) * config.NEWBIE_BASE_MEAN
+                             + self.player.numGames * mean) / config.NEWBIE_MIN_GAMES
+            return (adjusted_mean, dev)
+
+        return self.unadjusted_rating
+
     @property
     def rating(self):
+        num_games = self.player.numGames
```

Comment:
why define extra variable?

Reply:
CPython compiles code into pyc, it doesn't has JIT compiler.